src/record/export/usecases/get.ts

```diff
@@ -44,8 +49,15 @@ export const getRecords = async (
           attachmentsDir,
         }),
     );
+    numOfProcessedRecords += localRecords.length;
     await writer.write(localRecords);
   }
+
+  if (numOfProcessedRecords === 0) {
+    logger.warn(NO_RECORDS_WARNING);
+    return;
```

Comment:
MUST: Please remove `return` statement.
Even if we don't write anything to the writer, we need to close it by `writer.end()`.